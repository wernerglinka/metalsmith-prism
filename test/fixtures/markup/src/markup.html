<code class="language-markup">
<div class="panel panel-default">
  <div class="panel-body">
    Basic panel example
  </div>
</div>
</code>
