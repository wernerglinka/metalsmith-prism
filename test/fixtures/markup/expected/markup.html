<html><head></head><body><code class="language-markup">
  <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>div</span> <span class="token attr-name">class</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>panel panel-default<span class="token punctuation">"</span></span><span class="token punctuation">&gt;</span></span>
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;</span>div</span> <span class="token attr-name">class</span><span class="token attr-value"><span class="token punctuation attr-equals">=</span><span class="token punctuation">"</span>panel-body<span class="token punctuation">"</span></span><span class="token punctuation">&gt;</span></span>
      Basic panel example
    <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>div</span><span class="token punctuation">&gt;</span></span>
  <span class="token tag"><span class="token tag"><span class="token punctuation">&lt;/</span>div</span><span class="token punctuation">&gt;</span></span>
  </code>
  </body></html>
